### DISCUSSION TITLE
How can I target x86 on Windows?

### DISCUSSION THREAD

**Discussion by Amaracs** [nmoinvaz/minizip#595] [Q&A] [answered]
Hi all,

Sorry for my stupid question, but I can not figure it out.

How can I target it to 32bit windows?

I am doing the following:
`cmake . -DMZ_BUILD_TESTS=OFF -DMZ_BZIP2=OFF -DMZ_LZMA=OFF -DMZ_ZSTD=OFF -DMZ_SIGNING=OFF -DMZ_ICONV=OFF`
`cmake --build .`

Thank you in advance

**Comment by nmoinvaz** [comment, marked as answer]
Delete CMakeLists.txt and add "-A Win32" to your first cmake command.

**Reply by Amaracs** [reply]
Thanks : )

**Reply by Amaracs** [reply]
Also, is it possible to set the Runtime Library configuration to /MTd with cmake options, or it can be only set within the visual studio solution?

**Reply by nmoinvaz** [reply]
You can probably set it on the command line or in your own CMake project. Searching StackOverflow might give more information.

https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html
